database/migrations/Concerns/CanModifyPermissions.php

```diff
@@ -75,15 +75,29 @@ public function createPermissions(array $names, string $guardName): void
             ...$newGroups,
         ];
 
+        collect($guardName)
+            ->each(function (string $guardName) use ($names, $groups): void {
+                DB::table('permissions')
+                    ->insert(array_map(function (string $name, string $groupName) use ($groups, $guardName): array {
+                        return [
+                            'id' => (string) Str::orderedUuid(),
+                            'group_id' => $groups[$groupName],
+                            'guard_name' => $guardName,
+                            'name' => $name,
+                            'created_at' => now(),
+                        ];
+                    }, array_keys($names), array_values($names)));
+            });
```

Comment:
Though I understand the intention here from our conversation that these would not be run multiple times, let's add protections here to filter out permissions that already exist to prevent unique constraint violations in case something weird happens.